tests/CsvIteratorTest.php

```diff
@@ -0,0 +1,126 @@
+<?php
+
+namespace RonasIT\Support\Tests;
+use ErrorException;
+use ReflectionProperty;
+use RonasIT\Support\Iterators\CsvIterator;
+
+class CsvIteratorTest extends HelpersTestCase
+{
+    protected CsvIterator $csvIteratorClass;
+    protected ReflectionProperty $columnsProperty;
+
+    public function setUp(): void
+    {
+        parent::setUp();
+
+        $this->csvIteratorClass = new CsvIterator($this->getFixturePath('addresses.csv'));
+
+        $this->columnsProperty = new ReflectionProperty(CsvIterator::class, 'columns');
+        $this->columnsProperty->setAccessible(true);
+    }
+
+    public function testOpenFile()
+    {
+        $this->expectException(ErrorException::class);
+
+        $this->getMockBuilder(CsvIterator::class)
+            ->setConstructorArgs(['not_exists_file.csv'])
+            ->getMock();
```

Comment:
could we use regular class creation?

```suggestion
        new CsvIterator($this->getFixturePath('not_exists_file.csv'));
```

Reply:
Done